tests/integration/pox/pox-3/pooled-stacking-switch-pools.spec.ts

```diff
@@ -178,6 +180,84 @@ describe("testing pooled stacking under epoch 2.1", () => {
     );
 
     expect(poxAddrInfo0?.["total-ustx"]).toEqual(uintCV(90_000_000_000_000));
+
+    // Bob tries to lock 90m for Alice
+    response = await broadcastDelegateStackSTX(
+      {
+        poxVersion: 3,
+        network,
+        account: Accounts.WALLET_2,
+        fee: DEFAULT_FEE,
+        nonce: bobNonce++,
+      },
+      {
+        stacker: Accounts.WALLET_1,
+        amount: 90_000_000_000_000,
+        poolRewardAccount: Accounts.WALLET_2,
+        startBurnHeight: blockHeight,
+        lockPeriodCycles: 3,
+      }
+    );
+    expect(response.error).toBeUndefined();
+    [block, tx] = await waitForStacksTransaction(orchestrator, response.txid);
+    expect(tx.success).toBeFalsy();
+
+    // Bob tries to commit 90m
+    response = await broadcastStackAggregationCommitIndexed(
+      {
+        poxVersion: 3,
+        network,
+        account: Accounts.WALLET_2,
+        fee: DEFAULT_FEE,
+        nonce: bobNonce++,
+      },
+      { poolRewardAccount: Accounts.WALLET_2, cycleId: 2 }
+    );
+    expect(response.error).toBeUndefined();
+    [block, tx] = await waitForStacksTransaction(orchestrator, response.txid);
+    expect(tx.success).toBeFalsy();
```

Comment:
If tx failed, then it might be nice to validate the error.

Reply:
Good point. Added in 73ff762.